less/mixins.less

```diff
@@ -1 +1,32 @@
 // Mixins below variables
+
+.flex-jc-ai-center(@flex-dirctn:row;) {
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction:@flex-dirctn
+}
+
+.btn-creator(@BGcolor:@secondary-color) {
+    font-size:1.8rem;
+    text-align:center;
+    border-radius: 10px;
+    width:100px;
+    padding:15px;
+    color:white;
+    background-color:@BGcolor;
+
+    &:hover {
+        border:1px solid @secondary-color;
+        background-color: blue;
+        color:@secondary-color;
+    }
+}
+
+    .form-style(){
```

Comment:
The spacing here is a bit off. Consider using an extension like `Prettier` or `Beautify` to clean up your code.